builtin/logical/database/backend_test.go

```diff
@@ -64,7 +67,7 @@ func preparePostgresTestContainer(t *testing.T, s logical.Storage, b logical.Bac
 		})
 		if err != nil || (resp != nil && resp.IsError()) {
 			// It's likely not up and running yet, so return error and try again
-			return fmt.Errorf("err:%s resp:%#v\n", err, resp)
+			return errwrap.Wrapf(fmt.Sprintf("err:{{err}} resp:%#v", resp), err)
```

Comment:
Just a note that the whole "be consistent with errwrap" is generally much less necessary in package tests.

Reply:
Ah, although I see below where you do ContainsType so that's probably the root of this.